data/targets.yml

```diff
@@ -98,8 +98,8 @@
 :sle15sp6:
   obs_api: "https://api.suse.de/"
   obs_project: "Devel:YaST:SLE-15-SP6"
```

Comment:
Please add a comment that this change is just temporary and the "GA" needs to be later changed to "Update".